devscripts/run_tests.py

```diff
@@ -27,7 +27,6 @@ def parse_args():
 def run_tests(*tests, pattern=None, ci=False):
     run_core = 'core' in tests or (not pattern and not tests)
     run_download = 'download' in tests
```

Comment:
maybe these need to be more strict?

Reply:
Imo no, if theres `core` in the commandline itll apply markers. Theres 0 reason to use both markers and specify an explicit test. Using markers will still collect all tests, which then doesnt benefit `-k` as well